packages/libsdl2.rb

```diff
@@ -32,8 +32,4 @@ def self.build
       -Wno-dev"
     system "#{CREW_NINJA} -C builddir"
```

Comment:
We've converted to a buildsystem, right? Can't we drop this line and the rest of the self.build section?

Reply:
```ruby
@cmake_options = ARCH == 'x86_64' ? '-DOPT_DEF_SSEMATH=ON' : ''
```
wouldn't this work?

Reply:
Done.